MaxText/layers/mistral.py

```diff
@@ -117,21 +117,9 @@ def __call__(
     )
     intermediate_inputs = inputs + attention_lnx
 
-    # Fully Connected
-    hidden_states = rms_norm(
-        num_features=intermediate_inputs.shape[-1],
-        dtype=cfg.dtype,
-        weight_dtype=cfg.weight_dtype,
-        name="post_self_attention_layer_norm",
```

Comment:
Does this still work with existing checkpoints? I don't think name is being passed in anymore